modules/machineset-yaml-azure.adoc

```diff
@@ -18,29 +30,56 @@ kind: MachineSet
 metadata:
   labels:
     machine.openshift.io/cluster-api-cluster: <infrastructureID> <1>
+ifndef::infra[]
     machine.openshift.io/cluster-api-machine-role: <role> <2>
     machine.openshift.io/cluster-api-machine-type: <role> <2>
   name: <infrastructureID>-<role>-<region> <3>
+endif::infra[]
+ifdef::infra[]
+    machine.openshift.io/cluster-api-machine-role: infra <2>
+    machine.openshift.io/cluster-api-machine-type: infra <2>
+  name: <infrastructureID>-infra-<region> <3>
+endif::infra[]
   namespace: openshift-machine-api
 spec:
   replicas: 1
   selector:
     matchLabels:
       machine.openshift.io/cluster-api-cluster: <infrastructureID> <1>
+ifndef::infra[]
       machine.openshift.io/cluster-api-machineset: <infrastructureID>-<role>-<region> <3>
+endif::infra[]
+ifdef::infra[]
+      machine.openshift.io/cluster-api-machineset: <infrastructureID>-infra-<region> <3>
+endif::infra[]
   template:
     metadata:
       creationTimestamp: null
       labels:
         machine.openshift.io/cluster-api-cluster: <infrastructureID> <1>
+        ifndef::infra[]
         machine.openshift.io/cluster-api-machine-role: <role> <2>
         machine.openshift.io/cluster-api-machine-type: <role> <2>
         machine.openshift.io/cluster-api-machineset: <infrastructureID>-<role>-<region> <3>
```

Comment:
should be deleted

Reply:
Pointing out that this comment isn't marked as resolved.